roles/user-management/manage-atlassian-users/tasks/main.yml

```diff
@@ -6,20 +6,20 @@
   when: atlassian.groups|length > 0
```

Comment:
PR #295 changes these aspects of the `manage-atlassian-users` role, so if you could remove these changes from your PR it would make it easier on the merge.